com.unity.render-pipelines.universal/ShaderLibrary/SSAO.hlsl

```diff
@@ -17,8 +17,7 @@ SAMPLER(sampler_ScreenSpaceOcclusionTexture);
 // Params
 float4 _BlurOffset;
 float4 _SSAOParams;
-float4 _BaseMap_TexelSize;
-float4 _CameraDepthTexture_TexelSize;
+float4 _SourceSize;
```

Comment:
Just wondering, would it make sense to add this to UnityInput.hlsl and add a function to retrieve it in ShaderVariablesFunctions.hlsl ?

Reply:
I think we can revisit this once RTHandle will be implemented and If @phi-lira decides to add _ScreenSize property in functions form, that we talked about int one of the threads. Now I think there is no point to standardize it as this is not complete solution.

Reply:
Sounds good 👍